SplunkRumWorkspace/SplunkRum/SplunkRum/AppLifecycle.swift

```diff
@@ -19,6 +19,10 @@ import Foundation
 import UIKit
 import OpenTelemetrySdk
 
+let INACTIVITY_SESSION_TIMEOUT_SECONDS = 15 * 60
+private var sessionIdInActivityExpiration = Date().addingTimeInterval(TimeInterval(INACTIVITY_SESSION_TIMEOUT_SECONDS))
+var Is_New_SessionID_For_In_Activity = false
```

Comment:
Can we avoid random naked globals perhaps? Also the naming/casing should be consistent with other variables. P.S.: `inactivity`

Reply:
Yes,I will change name of `Is_New_SessionID_For_In_Activity` with `Is_New_SessionID_For_InActivity`.
Is_New_SessionID_For_In_Activity variable used to communicate between two files AppLifeCycle and Session. that's  why I make it global.